frontend/src/app/projects/models/diagrams/model-diagram-dialog/model-diagram-dialog.component.css

```diff
@@ -26,6 +26,25 @@
   flex-basis: calc(30% - 20px);
 }
 
+.copy-button {
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  padding: 5px 10px;
+  background-color: transparent;
+  color: #fff;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+  box-shadow: none;
+}
+
+.insert-button {
+  align-self: flex-end;
+  margin-top: auto;
+  color: white;
+}
```

Comment:
A while ago we added [tailwindcss](https://tailwindcss.com/) to the collab manager. Tailwind has the huge advantage that one can directly define styles in the `class` attribute and in a way shorter form. For example, instead of having `position: absolute` in the style attribute or css, one simply needs to use `class = "absolute"`.
In case we have a class that applies to several elements, using a css class is still a good choice, but in these two cases where both are single buttons, I would transform these rules into tailwind classes.

Reply:
I tried to replace it with tailwind classes but it does not seem to work completely yet which is why I will leave the css here for the moment